### PR TITLE
List of Sidebar Tabs

### PR DESCRIPTION
**Pull request by azooz2003-bit**
Self-explanatory. (e.g. Home, Assignments, Quizzes)

**Displaying Tabs**
This is for future explorations / requirements, but my approach for rendering tab pages would be as follows:
1. Retrieve URL to tab page.
2. Redirect using `WKWebview`. 
3. Inject OAuth2 token into webview as Cookie.

https://github.com/user-attachments/assets/cd9ba499-6618-41b6-8f41-a3851bad1c23

### CODE REVIEW

**Review by rahulon12 — COMMENTED** [review]
This LGTM, could you attach a quick recording?

Also not sure why the reordering of the `project.pbxproj` is happening - feel like this will later cause annoying merge conflicts.

File: CanvasPlusPlayground/Services/CanvasService.swift [hunk 1]
```diff
@@ -30,4 +30,6 @@ struct CanvasService {
             return nil
         }
     }
+    
+    // TODO: new method + dispatch queue for multiple concurrent requests - Aziz
```
Comment (rahulon12) [inline CanvasPlusPlayground/Services/CanvasService.swift:34]: Maybe create an issue and reference in the code here?

**Comment by azooz2003-bit** [comment]
@rahulon12 We can write a script that sorts the files in `pbxproj`. Probs overkill.

**Review by rahulon12 — APPROVED** [review]
Nice! 

Could you make an issue to open the specific page when tapping on the page itself? That can be worked on later, as you mentioned:

> Retrieve URL to tab page.
> Redirect using WKWebview.
> Inject OAuth2 token into webview as Cookie.

**Pull request merged**